src/Orchard.Web/Modules/Orchard.Users/Extensions/MembershipSettingsExtensions.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Orchard.Security {
+    public static class MembershipSettingsExtensions {
+        public static int GetMinimumPasswordLength(this IMembershipSettings membershipSettings) {
+            return membershipSettings.EnableCustomPasswordPolicy && membershipSettings.EnableCustomPasswordLength ? membershipSettings.MinimumPasswordLength : 1;
```

Comment:
And do we want to force an universal rule of a minimal password length now with these configuration options? E.g. if you want to use a four-digit PIN for your grandmother for the family website shouldn't this be your decision?

I mean ideally we'd hard-code a 30 character minimum limit but IMO it should be up to a website's maintainer to decide their password policy, including allowing weaker passwords, and now they can freely configure this.